src/global.js

```diff
@@ -122,19 +122,27 @@ export function getDisableSetCookie(): boolean {
   return false;
 }
 
+// TODO first-render-experiment-cleanup any thing using this function should use getFirstRenderExperiments
 export function getExperimentation(): Experimentation | null {
   if (typeof __EXPERIMENTATION__ !== "undefined") {
     if (__EXPERIMENTATION__) {
-      const experimation: Experimentation = {};
+      const experimentation: Experimentation = {};
       if (__EXPERIMENTATION__.__EXPERIENCE__) {
-        experimation.experience = __EXPERIMENTATION__.__EXPERIENCE__;
+        experimentation.experience = __EXPERIMENTATION__.__EXPERIENCE__;
       }
       if (__EXPERIMENTATION__.__TREATMENT__) {
-        experimation.treatment = __EXPERIMENTATION__.__TREATMENT__;
+        experimentation.treatment = __EXPERIMENTATION__.__TREATMENT__;
       }
-      return experimation;
+      return experimentation;
     }
   }
 
   return null;
 }
+
+export function getFirstRenderExperiments(): FirstRenderExperiments | null {
+  if (typeof __FIRST_RENDER_EXPERIMENTS__ !== "undefined") {
+    return __FIRST_RENDER_EXPERIMENTS__;
+  }
+  return null;
```

Comment:
could this return `{}` so that the code using it won't have to null check?

They'll have to see if their key was included regardless, this would take away checking if the object was null too